tags/interceptors_test.go

```diff
@@ -171,8 +194,9 @@ func (s *ClientStreamedTaggingSuite) TestPingStream_WithCustomTagsFirstRequest()
 
 		tags := tagsFromJson(s.T(), resp.Value)
 		assert.Contains(s.T(), tags, "peer.address", "the tags should contain key address")
+		assert.Contains(s.T(), tags, "grpc.request.deadline", "the tags should contain a deadline")
 		assert.Equal(s.T(), tags["grpc.request.value"], "something", "the tags should contain key address")
```

Comment:
`key address` in error message here looks wrong, should it be `grpc.request.value key`?